source/partials/next_release/_release.md.erb

```diff
@@ -1,51 +1,19 @@
-<h4>Security Fixes</h4>
+<h4>Bundled Plugins</h4>
 
-This release has critical security fixes to prevent CSRF, XSS and Session hijacking exploits. We highly recommend users to upgrade to keep your GoCD server and agent secure.
-Thanks [4cad](https://hackerone.com/4cad) for reporting these issues. 
+With this release GoCD provides support for [Pipleine as code](https://docs.gocd.org/current/advanced_usage/pipelines_as_code.html) through [JSON config repo plugin](https://github.com/tomzo/gocd-json-config-plugin) and the [YAML config repo plugin](https://github.com/tomzo/gocd-yaml-config-plugin). These plugins maintained by our core contributor [Tomasz Sętkowski](https://github.com/tomzo) will be bundled by default.
```

Comment:
Out of the Box 'Pipeline as code' support

With this release, GoCD supports [Pipeline as code](https://docs.gocd.org/current/advanced_usage/pipelines_as_code.html), out of the box. We've bundled the [JSON config repo plugin](https://github.com/tomzo/gocd-json-config-plugin) and the [YAML config repo plugin](https://github.com/tomzo/gocd-yaml-config-plugin) with the GoCD server. 
A shout out to our core contributor [Tomasz Sętkowski](https://github.com/tomzo), for maintaining these plugins.